test/test_playbooks/server_vars.yml.example

```diff
@@ -8,6 +8,12 @@ foreman_validate_certs: false
 # Parameter for domain test
 domain_dns_proxy: "centos7-devel.example.com"
 
+# Parameter for snapshot test
+snapshot_host_name: "test_host"
```

Comment:
Does the snapshot hostname belong into this PR?

Reply:
Probably not. ;)